fe/fe-core/src/main/java/com/starrocks/leader/LeaderImpl.java

```diff
@@ -1121,7 +1120,7 @@ private static void initTabletMeta(Tablet tablet, TTabletMeta tTabletMeta) {
     @NotNull
     private static List<TBackendMeta> getBackendMetas() {
         List<TBackendMeta> backends = new ArrayList<>();
-        for (Backend backend : GlobalStateMgr.getCurrentState().getNodeMgr().getClusterInfo().getBackends()) {
+        GlobalStateMgr.getCurrentState().getNodeMgr().getClusterInfo().backendAndComputeNodeStream().forEach(backend -> {
```

Comment:
Better to use variable `node` not `backend`

Reply:
done